figure.html

```diff
@@ -246,7 +246,7 @@
 </div> <!-- end #line-block -->
 
 <li>
-<p class="first">Click on the crop icon to open the <span class="bold">Crop to Region</span> panel.</p>
+<p class="first">Click on the <span class="bold">Crop</span> icon to open the <span class="bold">Crop to Region</span> panel.</p>
```

Comment:
Strictly this is no longer a "Crop icon". The icon has gone and it's now just a button called "Crop".